frontend/projects/ui/src/app/services/marketplace.service.ts

```diff
@@ -55,6 +56,20 @@ export class MarketplaceService implements AbstractMarketplaceService {
       }),
     )
 
+  private readonly filteredKnownHosts: Observable<StoreIdentity[]> =
+    combineLatest([
+      this.clientStorageService.showDevTools$,
+      this.knownHosts$,
+    ]).pipe(
+      map(([devMode, knownHosts]) => {
```

Comment:
I would use ternary operator and drop brackets and `return` keyword here and in `filteredMarketplace$`, but that's just my preferential code style:
```ts
map(([devMode, knownHosts]) => devMode
  ? knownHosts
  : knownHosts.filter(...)
),
```